doc/Data Plane Code Generation/Add_ProtocolMethods_In_HLCGenerated_SDK.md

```diff
@@ -0,0 +1,310 @@
+# Add Protocol Methods In HLC Generated SDK
+
+In this tutorial, we will step through the process of generating protocol methods in HLC SDK. Protocol methods will be generated in internal RestClient based on the `protocol-method-list` config.
+
+## Prerequisites
+
+You should have HLC SDK generated in `azure-sdk-for-net/sdk/<service name>/<package name>/src/Generated` folder.
+
+## Add protocol methods in generated RestClient
+
+In order to generate protocol methods in HLC generated RestClient, you'd have to define a `protocol-method-list` config in your `autorest.md` file. You can find `autorest.md` file path in `azure-sdk-for-net/sdk/<service name>/<package name>/src/autorest.md`. This config takes a list of full `operationId` defined in it's swagger definition. 
```

Comment:
It would be good to illustrate a short example here, consider something like:

An example of this configuration is:
```
protocol-method-list:
  - OperationGroup1_OperationName1
  - OperationGroup2_OperationName1
  - ```

Reply:
I've added an example here: https://github.com/ShivangiReja/azure-sdk-for-net/blob/AddDocument/doc/Data%20Plane%20Code%20Generation/Add_ProtocolMethods_In_HLCGenerated_SDK.md#add-config. If you click Details of (autorest.md). I've added -

````
### Generate DPG methods
```yaml
protocol-method-list:
  - Table_Delete
  - Service_SetProperties
```
````

Reply:
Updated!